<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QWidget" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>256</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;
background: transparent;s
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>40</y>
     <width>256</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;
background: transparent;s
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>974</x>
     <y>40</y>
     <width>256</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;
background: transparent;s
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>17</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>Game 1 Scores:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>17</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>Game 2 Scores:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>16</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>Game 3 Scores:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>270</y>
     <width>51</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {
background: rgb(140, 0, 105);
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}
QProgressBar{
text-align: bottom;
font: 18pt &quot;Digital-7&quot;;
background: transparent;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>270</y>
     <width>51</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {
background: rgb(140, 0, 105);
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}
QProgressBar{
text-align: bottom;
font: 18pt &quot;Digital-7&quot;;
background: transparent;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_3">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>270</y>
     <width>51</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {
background: rgb(140, 0, 105);
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}
QProgressBar{
text-align: bottom;
font: 18pt &quot;Digital-7&quot;;
background: transparent;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_4">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>270</y>
     <width>51</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {
background: rgb(140, 0, 105);
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}
QProgressBar{
text-align: bottom;
font: 18pt &quot;Digital-7&quot;;
background: transparent;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_5">
   <property name="geometry">
    <rect>
     <x>1025</x>
     <y>270</y>
     <width>51</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {
background: rgb(140, 0, 105);
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}
QProgressBar{
text-align: bottom;
font: 18pt &quot;Digital-7&quot;;
background: transparent;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_6">
   <property name="geometry">
    <rect>
     <x>1135</x>
     <y>270</y>
     <width>51</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {
background: rgb(140, 0, 105);
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}
QProgressBar{
text-align: bottom;
font: 18pt &quot;Digital-7&quot;;
background: transparent;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>570</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>Game 1 Performance</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>570</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>Game 2 Performance</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>570</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>Game 3 Performance</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>530</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>Your average</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>530</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>Global average</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>530</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>Global average</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>530</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>Your average</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1095</x>
     <y>530</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>Global average</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>985</x>
     <y>530</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>Your average</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
